### ISSUE TITLE
Treat "null" and "undefined" as interchangeable in query params interface

### ISSUE THREAD

**Issue by lucaswoj** [adelsz/pgtyped#36]
If I have a value that's of type `string | undefined` (for example) I need to coerce it to `string | null` before I can pass it as a query param. 

```tsx
const accessToken: string | undefined = urlParameters.accessToken;
await isAccessTokenValidForFeed.run({
    accessToken: accessToken || null,
    feedId
}, db);
```

It's true that SQL has very different `null` semantics than JS but this shouldn't matter for input params. 

(Again, happy to put together a PR for this if its a desirable feature.)

**Comment by adelsz** [comment]
Agree, this is a good idea 👍 
`node-postgres` and `pg-promise` support autocasting `undefined` to `null` when passed as an input param.
Sure, a PR is welcome, let me know if you will need any help there.

**Comment by adelsz** [comment]
Fixed in v0.6.0!
